src/classes/GE_GiftEntryController.cls

```diff
@@ -40,6 +40,10 @@ public with sharing class GE_GiftEntryController {
     public static final String PURCHASE_CALL_TIMEOUT_MESSAGE = System.Label.geErrorRequestTimedout;
     private static final String BOOLEAN_DATA_TYPE = 'BOOLEAN';
 
+    @AuraEnabled(cacheable=true)
+    public static Map<String, Integer> getGiftBatchTotalsBy(String batchId) {
+        return new GiftBatchService().getGiftBatchTotalsMapFor((Id) batchId);
```

Comment:
Can Gift Batch Service be an injectable class level variable instead of newing up an instance here?